### PR TITLE
Fixes for call UI

### PR DESCRIPTION
**Pull request by dbkr**
Separated by commit

 * Add bottom margin on PiP view
 * Remove avatar blurring & pause icon for held calls
 * Change background of incoming call box to match PiP view
 * Put drop shadow & border radius on PiP view & incoming call box
   rather than the CallContainer they're in (so they each have their own
   drop shadow / rounded corners).
 * Add margin between incoming call box and PiP view
 * Use 'Switch' as resume link if unholding would hold another call
 * Remove the secondary call avatar
 * Change 'paused' to 'on hold'

### Avatar no longer blurred
Screenshot 2020-12-18 at 15 51 16

### Margin between PiP View & Incoming Call
Screenshot 2020-12-18 at 18 07 18

### Secondary avatar removed, 'paused' changed to 'on hold'
Screenshot 2020-12-18 at 19 53 15

### 'Switch' link in PiP view
Screenshot 2020-12-18 at 19 40 29

### '[Switch](https://matrix.fandom.com/wiki/Switch)' link in large call view
Screenshot 2020-12-18 at 19 40 37

@nadonomy for approval

### CODE REVIEW

**Review by nadonomy — APPROVED** [review]
LGTM

File: res/css/views/voip/_CallView.scss [hunk 1]
```diff
@@ -84,6 +88,7 @@ limitations under the License.
         border-radius: 2000px;
         overflow: hidden;
         position: relative;
+        /* Blurred avatar images & pause icon for on-hold removed for now
```
Comment (jryans) [inline res/css/views/voip/_CallView.scss:91]: Would it make more sense to just remove it? Or you expect it to return soon...? It's always available in version control. Anyway, up to you.

File: src/CallHandler.tsx [hunk 2]
```diff
@@ -617,6 +617,18 @@ export default class CallHandler {
         }
     }
 
+    /**
+     * @returns true if we are currently in anu call where we haven't put the remote party on hold
```
Comment (jryans) [inline src/CallHandler.tsx:621]:
```suggestion
     * @returns true if we are currently in any call where we haven't put the remote party on hold
```

**Review by jryans — APPROVED** [review]
Looks good overall! 😄

File: res/css/views/voip/_CallView.scss (hunk 1 above)
Comment (dbkr) [reply res/css/views/voip/_CallView.scss]: Yeah true, otherwise it'll just hang around in the css for ages

**Pull request merged**